website/www/site/data/performance.yaml

```diff
@@ -0,0 +1,298 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+host: https://28cd0c7f-7a84-428c-bfa7-6e8316f6931b.looker.app
+path: looks
+looks:
+  bigquery:
+    read:
+      test_name:
+        - id: nwQxvsnQFdBPTk27pZYxjcGNm2rRfNJk
```

Comment:
it appears only a few graphs render successfully, e.g. http://apache-beam-website-pull-requests.storage.googleapis.com/27540/performance/bigquery/index.html

may just show "Runtime and Estimated Cost", "AvgIn/OutputThroughputBytesPerSec", "AvgIn/OutputThroughputElementsPerSec" plots that's all we need

Reply:
From staging http://apache-beam-website-pull-requests.storage.googleapis.com/27540/performance/bigtable/index.html now there is still 20+ plots shown. Please just show "Runtime and Estimated Cost", "AvgIn/OutputThroughputBytesPerSec", "AvgIn/OutputThroughputElementsPerSec" plots that's all we need (for batch)

Reply:
@damondouglas can we just show what @Abacn list for now?

Reply:
Per our discussion in chat I will resolve this.